ros_ign_bridge/test/utils/ign_test_msg.hpp

```diff
@@ -40,14 +40,17 @@
 #include <ignition/msgs/magnetometer.pb.h>
 #include <ignition/msgs/model.pb.h>
 #include <ignition/msgs/odometry.pb.h>
+#include <ignition/msgs/odometry_with_covariance.pb.h>
 #include <ignition/msgs/pointcloud_packed.pb.h>
 #include <ignition/msgs/pose.pb.h>
+#include <ignition/msgs/pose_with_covariance.pb.h>
 #include <ignition/msgs/pose_v.pb.h>
 #include <ignition/msgs/quaternion.pb.h>
 #include <ignition/msgs/stringmsg.pb.h>
 #include <ignition/msgs/stringmsg_v.pb.h>
 #include <ignition/msgs/track_visual.pb.h>
 #include <ignition/msgs/twist.pb.h>
+#include <ignition/msgs/twist_with_covariance.pb.h>
```

Comment:
We've included TwistWithCovariance, but I don't see any additions using it below. Should we add something?

Reply:
oh, I see your TODO now :)

Reply:
Added tests for both twist and odometry with covariance.